<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>&gt;</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="autoRepeatInterval">
      <number>20</number>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="autoRepeatInterval">
      <number>20</number>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>HOME</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QStackedWidget" name="monitors">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_1">
      <layout class="QHBoxLayout" name="gaugeLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="PlotWidget" name="graph"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QSpinBox" name="currentPositionBox">
     <property name="minimum">
      <number>-10000</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="6" rowspan="6">
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/control/plus.svg</normaloff>:/control/plus.svg</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/control/play.svg</normaloff>:/control/play.svg</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="coilFreeButton">
        <property name="text">
         <string>Free Coils</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="spinBox_2">
     <property name="maximum">
      <number>20000</number>
     </property>
     <property name="value">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QComboBox" name="pinSet">
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Steps per full revolution</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <layout class="QHBoxLayout" name="configLayout"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>currentPositionBox</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>pinSet</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>graph</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>initialize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>stepLeft()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>stepRight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentPositionBox</sender>
   <signal>editingFinished()</signal>
   <receiver>Dialog</receiver>
   <slot>stepTo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pinSet</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>setPins(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>coilFreeButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>freeCoils(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add()</slot>
  <slot>play()</slot>
  <slot>initialize()</slot>
  <slot>stepLeft()</slot>
  <slot>stepRight()</slot>
  <slot>stepTo()</slot>
  <slot>setPins(int)</slot>
  <slot>freeCoils(bool)</slot>
 </slots>
</ui>
